exporter/resource_exporter.go

```diff
@@ -62,11 +74,17 @@ func NewResourceExporter(ctx context.Context, schema *ResourceSchema, reg *prom.
 }
 
 func (r *ResourceExporter) ExportMetrics(ctx context.Context) error {
-	// FIXME
-	timer := time.NewTicker(time.Second * 5)
+	timer := time.NewTicker(r.collectionInterval)
+
+	firstRun := make(chan struct{}, 1)
+	firstRun <- struct{}{}
```

Comment:
I think we can just ` timer.C <- struct{}{}` to trigger the instant startup and avoid another case?

Reply:
Unfortunately not, `timer.C` is a receive-only channel.